runtime/src/main/java/org/corfudb/runtime/collections/streaming/LRStreamingTask.java

```diff
@@ -0,0 +1,62 @@
+package org.corfudb.runtime.collections.streaming;
+
+import com.google.common.base.Preconditions;
+import com.google.protobuf.Message;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.collections.StreamListener;
+import org.corfudb.runtime.collections.Table;
+import org.corfudb.runtime.collections.TableSchema;
+import org.corfudb.runtime.view.TableRegistry;
+import javax.annotation.Nonnull;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.ExecutorService;
+
+public class LRStreamingTask<K extends Message, V extends Message, M extends Message> extends StreamingTask {
```

Comment:
A basic one: Could you please add file level comments and an example how it will be consumed?